app/errors.js

```diff
@@ -8,3 +8,6 @@ exports.databaseError = message => internalError(message, exports.DATABASE_ERROR
 
 exports.DEFAULT_ERROR = 'default_error';
 exports.defaultError = message => internalError(message, exports.DEFAULT_ERROR);
+
+exports.SCHEMA_VALIDATION = 'schema_validation';
+exports.schemaValiation = errors => internalError(errors, exports.SCHEMA_VALIDATION);
```

Comment:
I know its a bit wordy, but generally error functions should have de keyword `error in their names` it makes the code easier to read. It should be `schemaValidationError`, the same goes for `SCHEMA_VALIDATION_ERROR`